pkgs/development/python-modules/dvclive/default.nix

```diff
@@ -5,13 +5,13 @@
 , pytestCheckHook
 , pythonOlder
 , ruamel-yaml
-, setuptools
+, setuptools-scm
 , tabulate
 }:
 
 buildPythonPackage rec {
   pname = "dvclive";
-  version = "1.3.0";
+  version = "1.2.2";
```

Comment:
Instead of downgrading the module the constraint of the consumer should be relaxed.

Reply:
Oops, careless on my part. Internalized it as 1.2.2 -> 1.3.0 despite clearly doing the opposite :suspect: The `setuptools-scm` change is still required to provide the right version information though.

Reply:
DVC is /very/ sensitive to its package versions, relaxing constraints have broken things for alternative backends multiple times. Please consider including the exact versions they specify as much as possible.

Reply:
dvc specifies `dvclive>=1.2.2` so no need to relax, this was just dumb on my part.